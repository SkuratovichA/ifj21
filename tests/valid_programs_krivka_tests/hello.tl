-- Hello World example in IFJ21
-- run it on Merlin by: tl run hello.tl

require "ifj21"

function hlavni_program(year : integer)
  write("Hello from IFJ", year, "\n")
end

hlavni_program(21)
hlavni_program(22) -- pozdrav z budoucnosti
