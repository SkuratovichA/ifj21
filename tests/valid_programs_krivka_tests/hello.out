Hello from IFJ21
Hello from IFJ22
